x11-terms/ptyxis/ptyxis-46.3.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="Ptyxis is a terminal for GNOME with first-class support for containers."
+HOMEPAGE="https://gitlab.gnome.org/chergert/ptyxis"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~loong"
```

Comment:
If you only tested it on amd64, please drop the other KEYWORDS

Reply:
Done in https://github.com/gentoo/gentoo/pull/34844/commits/50c97d53444bad308d41d3a12616af34fc17a1b3